apps/src/util/experiments.js

```diff
@@ -45,15 +47,11 @@ experiments.getEnabledExperiments = function () {
   return this.getStoredExperiments_().map(experiment => experiment.key);
 };
 
-experiments.setEnabled = function (key, shouldEnable) {
+experiments.setEnabled = function (key, shouldEnable, expiration) {
```

Comment:
should this be `expiration=null` or something to make it clear the param is optional?